sormas-api/src/main/resources/strings.properties

```diff
@@ -580,6 +580,7 @@ messageCreateContactCase=The final laboratory result of the contact the sample a
 messageDatabaseExportFailed = Please contact an admin and notify them about this problem
 messageEntryCreated = Entry created
 messageEpiDataHint = <i>Please indicate if any of the following is relevant for the patient during the incubation period.</i>
+pseudonymisedCasesSelectedWarning =<p> For of the bulked edited cases you have only limited access to the sensitive data.</p><p> For those cases the value you put into "Place Description" will be ignored.</p>
```

Comment:
Some words seem to be missing here. Also remove the blank space after the `<p>` tag.
Additionally: "bulked edited" => "bulk-edited"